tracer/build/_build/CodeGenerators/TargetFrameworks.cs

```diff
@@ -27,3 +27,12 @@ internal enum TargetFrameworks : uint
     NET8_0 = 512,
     NET0_0 = 1024,
 }
+
+
+internal static class TargetFrameworksExtensions
+{ 
+    public static bool IsNetFxOnly(this TargetFrameworks tfm)
+    {
+        return ((uint)(tfm & (~(TargetFrameworks.NET461 | TargetFrameworks.NET462)))) == 0;
```

Comment:
_Technically_ this isn't correct, e.g.:
```suggestion
        return tfm != TargetFrameworks.None
        && ((uint)(tfm & (~(TargetFrameworks.NET461 | TargetFrameworks.NET462)))) == 0;
```

Reply:
But tbh, I would be inclined to do this 😄 
```suggestion
        return tfm == TargetFrameworks.NET461 
            || tfm == TargetFrameworks.NET462 
            || tfm == (TargetFrameworks.NET461 | TargetFrameworks.NET462);
```

In fact, why aren't we _only_ defining the values that this _can_ be (i.e. our Datadog.Trace.dll TFMs 🤔) i.e. 

```csharp
internal enum TargetFrameworks : uint
{
    None = 0, // For CodeGenerator only
    NET462 = 1,
    NETSTANDARD2_0 = 2,
    NETCOREAPP3_1 = 4,
    NET6_0 = 8,
}
```